lib/internal/event_target.js

```diff
@@ -304,7 +304,7 @@ class Event {
   set cancelBubble(value) {
     if (!isEvent(this))
       throw new ERR_INVALID_THIS('Event');
-    if (value) {
+    if (value === false) {
       this.stopPropagation();
     }
```

Comment:
@mertcanaltin I think current modification causes failure of test case because `cancelBubble` is alias of `stopPropagation()` when I checked spec(https://dom.spec.whatwg.org/#dom-event-cancelbubble).

I think original code is fine, but if it has to be changed, below code is more proper.
```suggestion
    if (value) {
      this.#propagationStopped = true;
    }
```
@KhafraDev @benjamingr wdyt?

Reply:
thank you very much for your suggestion
this seems to be a better solution, I am applying it